sagemaker-pipeline-compare-model-versions/README.md

```diff
@@ -0,0 +1,23 @@
+# Comparing model metrics with SageMaker Pipelines and SageMaker Model Registry
+
+### SageMaker Model Registry
+SageMaker Model Registry is a central location where customers can manage their machine learning models, compare different model versions and visualise metrics. It's a registry with which data scientists can register machine learning models with relevant meta data and metrics, and from which machine learning engineers or DevOps engineers can deploy machine learning models. In larger MLOps systems, a model registry is usually where the teams in charge of deploying machine learning models, meet the teams in charge of developing and training machine learning models.
+
+![Compare versions](images/compare-versions.png "Compare versions")
+
+
+### MLOps
+
+MLOps, or Machine Learning Operations, is the concept of applying DevOps practices on the lifecyle of a machine learning model. It's the process of creating automated workflows that enable organisations to iterate on machine learning models after they have been deployed into a production environment. MLOps usually consists of two workflows that sits on either side of a machine learning model registry; one to train a model and one to deploy a model. A model registry is a central location to manage machine learning models, where ML engineers and data scientists can compare different model versions, visualize metrics, and decide which versions to accept and which to reject. Ideally, approving a new version of a model triggers a pipeline that ultimately deploys the model into production.
+
+
+From a high-level perspective, it can look like this.
+
+![High-level MLOps](images/high-level.png "MLOps from a high-level")
+
+The deployment pipeline is what is most similar to traditional CI/CD methods, propagating an artifact through a pipeline that performs testing and if the tests are successful, deploys them. Anny CI/CD tools can be used for the deployment, but to automate the entire machine learning model lifecycle, a pipeline on "the other side" of the model registry is requireed as well. Another reusable pipeline through which new data can be pushed when monitoring identifies some form of data or model drift. A pipeline that ultimately produces a new machine learning model and registers it with the model registry. A SageMaker Pipeline.
```

Comment:
Anny, typo

Reply:
"Another reusable pipeline through which new data can be pushed when monitoring identifies some form of data or model drift." 
since we are describing this part, can we drop this?